enhancements/console/customize-perspectives.md

```diff
@@ -0,0 +1,294 @@
+---
+title: customize-perspectives
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-26
+last-updated: 2022-08-25
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Perspectives 
+
+## Summary
+
+OpenShift Container Platform (OCP) web console provides two perspectives: Administrator and Developer. 
+The Developer perspective provides workflows specific to developer use cases 
+like create, deploy and monitor applications, while Administrator perspective 
+is responsible for managing the cluster resources, users, and projects.
+
+This document provides details about how cluster administrator 
+can hide user perspectives. 
+
+![Perspectives switch dropdown](perspectives-switch.png)
+
+## Motivation
+
+Cluster administrator wants to hide the "Administrator" perspective 
+from non-privileged and to hide the "Developer" 
+perspective for all users.
+
+### User Stories
+
+#### Story 1
+
+As a cluster admin, I want to hide "Administrator" perspective 
+from all non-privileged users so that non-privileged users
+cannot access "Administrator" perspective for managing the cluster resources, 
+users, and projects. Non-privileged users are users that cannot get
```

Comment:
Strikes me as we want to hide this to avoid confusion? They could see that stuff but they don't have RBAC for it so couldn't interact with it, perhaps we can expand the why of this to include this idea?

Reply:
Yes, it's correct. User cannot break anything since we do RBAC check, we hide or disable buttons etc , we still get the feedback from customers that Administrator perspective is confusing for some developers. 

Currently all users have access to all perspectives and this might confuse some developers.

New perspectives could be added by the customer so they want to hide the default perspectives.

Reply:
> Administrator perspective is confusing for some developers.

This is useful for the user stories section!